x-pack/legacy/plugins/siem/public/pages/network/ip_details.tsx

```diff
@@ -18,36 +18,37 @@ import { HeaderPage } from '../../components/header_page';
 import { LastEventTime } from '../../components/last_event_time';
 import { getNetworkUrl } from '../../components/link_to/redirect_to_network';
 import { manageQuery } from '../../components/page/manage_query';
-import { DomainsTable } from '../../components/page/network/domains_table';
 import { FlowTargetSelectConnected } from '../../components/page/network/flow_target_select_connected';
 import { IpOverview } from '../../components/page/network/ip_overview';
 import { UsersTable } from '../../components/page/network/users_table';
 import { TlsTable } from '../../components/page/network/tls_table';
-import { DomainsQuery } from '../../containers/domains';
 import { GlobalTime } from '../../containers/global_time';
 import { IpOverviewQuery } from '../../containers/ip_overview';
 import { indicesExistOrDataTemporarilyUnavailable, WithSource } from '../../containers/source';
 import { TlsQuery } from '../../containers/tls';
 import { UsersQuery } from '../../containers/users';
-import { FlowTarget, LastEventIndexKey } from '../../graphql/types';
+import { FlowTarget, FlowTargetNew, LastEventIndexKey } from '../../graphql/types';
```

Comment:
Doesn't look like you added this type, but these sort of names with generic modifiers make it hard to grok the differentiation between the original type, well of course other than the temporal aspect of it, haha 🙂 

Perhaps `FlowTargetSourceDest` would be more fitting? Or `FlowTargetBasic`, `FlowTargetFiltered`? Naming is hard 😬